workspaces/services/guacamole/porter.yaml

```diff
@@ -0,0 +1,92 @@
+name: tre-service-guacamole
+version: 0.1.2
+description: "An Azure TRE service for Guacamole"
+registry: azuretre
+dockerfile: Dockerfile.tmpl
+
+credentials:
+  - name: azure_tenant_id
+    env: ARM_TENANT_ID
+  - name: azure_subscription_id
+    env: ARM_SUBSCRIPTION_ID
+  - name: azure_client_id
+    env: ARM_CLIENT_ID
+  - name: azure_client_secret
+    env: ARM_CLIENT_SECRET
+
+parameters:
+  - name: workspace_id
+    type: string
+  - name: tre_id
+    type: string
+  - name: acr_name
+    type: string
+    description: "Name of the azure container registry"
+    env: ACR_NAME
+  - name: mgmt_resource_group_name
+    type: string
+    description: "Resource group containing the shared ACR"
+    env: MGMT_RESOURCE_GROUP_NAME
+  - name: guacamole_image_tag
+    type: string
+    description: "Guacamole server image tag"
+    env: GUACAMOLE_IMAGE_TAG
+  - name: tfstate_resource_group_name
+    type: string
+    description: "Resource group containing the Terraform state storage account"
+  - name: tfstate_storage_account_name
+    type: string
+    description: "The name of the Terraform state storage account"
+  - name: tfstate_container_name
+    env: tfstate_container_name
+    type: string
+    default: "tfstate"
+    description: "The name of the Terraform state storage container"
```

Comment:
We need to ensure this works with managed identities:
```
  - name: arm_use_msi
    env: ARM_USE_MSI
    default: false
```
Challenge is cannot test locally with managed identities - lets have a chat you might have a way forward.